Documentation/running-clair.md

```diff
@@ -44,14 +44,15 @@ A [PostgreSQL 9.4+] database instance is required for all instructions.
 #### Kubernetes
 
 If you don't have a local Kubernetes cluster already, check out [minikube].
+This also requires [helm].
 
 [minikube]: https://github.com/kubernetes/minikube
+[helm]: https://github.com/kubernetes/helm
 
 ```
 git clone https://github.com/coreos/clair
-cd clair/contrib/k8s
-kubectl create secret generic clairsecret --from-file=./config.yaml
-kubectl create -f clair-kubernetes.yaml
+cd clair/contrib/helm
+helm install clair
```

Comment:
we should probably instruct users to create their own `values.yaml`:

```
cd clair/contrib/helm
cp values.yaml ~/my_custom_values.yaml
vi ~/my_custom_values.yaml
helm install clair -f ~/my_custom_values.yaml
```

Reply:
thanks for the quick feedback @jzelinskie